### ISSUE TITLE
[FE] 10-10EZ - Improvement: Update Introduction page to align with Design System guidance

### ISSUE THREAD

**Issue by hdjustice** [department-of-veterans-affairs/va.gov-team#75437]
## Benefit to the Veteran
By updating our web components and logic on the Introduction page, the Veteran will have a consistent and positive experience across VA.gov.  The existing alerts prompting the Veteran to sign-in to their VA.gov account is to allow their progress to be saved, in the event they need to exit the form, and also to avoid duplicate applications, if the Veteran has already applied for healthcare.

## Background
We received [feedback/GH Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69447) from CAIA team on our Introduction page stating that we were using an incorrect pattern and content.

There are comments back & forth, working with Aliyah with CAIA team to determine the best solution.  We will not edit or remove the existing alerts, and we will be updating to v3 USWDS web components before 2/16.

In addition to updating to V3 web components, we also need to update some of the logic to ensure the new web components display and work properly.

As a result of a late daily deployment which introduced a PR that impacted submissions and access to the 10-10EZ, We may want to explore the possibility that the 1010ez introduction page does not have a fall-through/default case in its conditional display logic, and figure out how best to address this.

## Resources
- [Figma File](https://www.figma.com/file/UljiHam46o5DItC5iDgmPd/10-10EZ?type=design&node-id=0-16200&mode=design&t=AVtz43N8ozIZviQ5-0)

## Tasks
- [x] Meet with UX to discuss various items to be re-worked
- [x] Outline the decisions made, including release plan
- [x] Implement per outline
- [x] Complete E2E testing
- [x] PM to validate in Staging
- [x] Release to prod per decision made during discussions

## Acceptance Criteria
- The Introduction page will be updated to align with the design system guidance.

**Comment by longmd** [comment]
@jessicastump , question around the FAQs for the various enrollment statuses. Since we went back and limited the status codes that could fill out an application, should we modify the FAQs for the ones that cannot fill out the application? 

See the following example:

<img width="487" alt="Screenshot 2024-03-12 at 14 09 26" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/6738544/38bfd787-599f-4483-b7a9-20eca61d1d5d">

We have that sentence about applying again if they've been told to do so, but we dont give them that option any longer.

**Comment by jessicastump** [comment]
@longmd We should look at this for each status just to be sure, but I think removing that last sentence/paragraph around recommend to apply again and leaving it with them calling HEC for those that don't have the "start new application" cta would work. 
Also curious if we have any data in GA or otherwise about how often these status pages come up...

**Comment by longmd** [comment]
@jessicastump , that sounds good. I will do the audit of that and remove where appropriate. And there will be no metrics because these aren't individual pages. We just render different content on the intro page based on the data we have.

**Comment by longmd** [comment]
Good news.... all good to go with the updates here. 

1. Regression testing came out perfect. 
2. We have additional automated testing for all possible authentication and enrollment statuses and their associated content or alert types to ensure we are showing the right content to the right use case per the Figma source of truth.
3. We have a full fall-through that was previously missing in the logic, so we should likely never see an occurrence like we saw in late Feb 2024.
4. We reduced a ton of bloat and complexity in the conditional logic to clean this part of the app up a ton (over 500 excess lines of code removed).
